Gemfile

```diff
@@ -20,34 +20,32 @@ gem 'jbuilder', '~> 2.7.0'
 gem 'nokogiri', '>= 1.8.2'
 gem 'puma', '~> 5.5.1'
 gem 'rack', '~> 2.2.3'
-gem 'rails', '~> 5.2'
+gem 'rails', '~> 6.1'
 gem 'rake'
-# gem 'sprockets', '~> 3.7.0', '>= 3.7.2'
 
 gem 'pg'
 gem 'sequel'
 gem 'sequel-pg_advisory_locking'
 gem 'sequel-postgres-schemata', require: false
 gem 'sequel-rails'
 
-gem 'activesupport'
+gem 'activesupport', '~> 6.1'
 gem 'base32-crockford'
-gem 'bcrypt', '~> 3.1.2'
+gem 'bcrypt'
 gem 'gli', require: false
+gem 'httpclient', github: 'nahi/httpclient', branch: 'master'
 gem 'listen'
-gem 'slosilo', '~> 2.2'
+gem 'slosilo', '~> 3.0.0'
 
 # Explicitly required as there are vulnerabilities in older versions
 gem "ffi", ">= 1.9.24"
 gem "loofah", ">= 2.2.3"
 
-# Installing ruby_dep 1.4.0
-# Gem::InstallError: ruby_dep requires Ruby version >= 2.2.5, ~> 2.2.
-gem 'ruby_dep', '= 1.3.1'
-
-gem 'conjur-api', '~> 5.pre'
+# Pinned to update for role member search, using ref so merging and removing
+# the branch doesn't immediately break this link
+gem 'conjur-api'
 gem 'conjur-policy-parser', path: 'gems/policy-parser'
-gem 'conjur-rack', '~> 4'
+gem 'conjur-rack', github: 'cyberark/conjur-rack', branch: 'bump-version-5'
```

Comment:
Switch to released version or remove constraint

Reply:
Fixed - remove constraint